core/aggregation.go

```diff
@@ -45,22 +50,32 @@ type SignatureAggregator interface {
 
 	// AggregateSignatures blocks until it recieves a response for each operator in the operator state via messageChan, and then returns the aggregated signature.
 	// If the aggregated signature is invalid, an error is returned.
-	AggregateSignatures(state *IndexedOperatorState, quorumIDs []QuorumID, message [32]byte, messageChan chan SignerMessage) (*SignatureAggregation, error)
+	AggregateSignatures(ctx context.Context, state *IndexedOperatorState, quorumIDs []QuorumID, message [32]byte, messageChan chan SignerMessage) (*SignatureAggregation, error)
```

Comment:
I wonder if it should be the responsibility of the signature aggregator to look up the operator addresses. 

We could send the `SignerMessage`s to a different component which adds the operator addresses before forwarding to whatever channel the SignatureAggregator is listening to. 

I don't have a strong opinion and the current way of doing it looks like it doesn't look like it causes any issues.